set.mm

```diff
@@ -659185,6 +659196,173 @@ D Fn ( I ... ( M - 1 ) ) /\
+
+    $( The identity element of a group is a right identity.  Deduction
+       associated with ~ grprid .  (Contributed by SN, 29-Jan-2025.) $)
+    sn-grpridd $p |- ( ph -> ( X .+ .0. ) = X ) $=
+      ( cgrp wcel co wceq grprid syl2anc ) ADLMEBMEFCNEOJKBCDEFGHIPQ $.
+  $}
+
+  ${
+    grpassd.b $e |- B = ( Base ` G ) $.
+    grpassd.p $e |- .+ = ( +g ` G ) $.
+    grpassd.g $e |- ( ph -> G e. Grp ) $.
+    grpassd.1 $e |- ( ph -> X e. B ) $.
+    grpassd.2 $e |- ( ph -> Y e. B ) $.
+    grpassd.3 $e |- ( ph -> Z e. B ) $.
+    $( A group operation is associative.  (Contributed by SN, 29-Jan-2025.) $)
+    grpassd $p |- ( ph -> ( ( X .+ Y ) .+ Z ) = ( X .+ ( Y .+ Z ) ) ) $=
+      ( cgrp wcel co wceq grpass syl13anc ) ADNOEBOFBOGBOEFCPGCPEFGCPCPQJKLMBCD
+      EFGHIRS $.
+  $}
+
+  ${
+    grpinvcld.b $e |- B = ( Base ` G ) $.
+    grpinvcld.n $e |- N = ( invg ` G ) $.
+    grpinvcld.g $e |- ( ph -> G e. Grp ) $.
+    grpinvcld.1 $e |- ( ph -> X e. B ) $.
+    $( A group element's inverse is a group element.  (Contributed by SN,
+       29-Jan-2025.) $)
+    grpinvcld $p |- ( ph -> ( N ` X ) e. B ) $=
+      ( cgrp wcel cfv grpinvcl syl2anc ) ACJKEBKEDLBKHIBCDEFGMN $.
+  $}
+
+  ${
+    grplinvd.b $e |- B = ( Base ` G ) $.
+    grplinvd.p $e |- .+ = ( +g ` G ) $.
+    grplinvd.u $e |- .0. = ( 0g ` G ) $.
+    grplinvd.n $e |- N = ( invg ` G ) $.
+    grplinvd.g $e |- ( ph -> G e. Grp ) $.
+    grplinvd.1 $e |- ( ph -> X e. B ) $.
+    $( The left inverse of a group element.  Deduction associated with
+       ~ grplinv .  (Contributed by SN, 29-Jan-2025.) $)
+    grplinvd $p |- ( ph -> ( ( N ` X ) .+ X ) = .0. ) $=
+      ( cgrp wcel cfv co wceq grplinv syl2anc ) ADNOFBOFEPFCQGRLMBCDEFGHIJKST
+      $.
+
+    $( The right inverse of a group element.  Deduction associated with
+       ~ grprinv .  (Contributed by SN, 29-Jan-2025.) $)
+    sn-grprinvd $p |- ( ph -> ( X .+ ( N ` X ) ) = .0. ) $=
+      ( cgrp wcel cfv co wceq grprinv syl2anc ) ADNOFBOFFEPCQGRLMBCDEFGHIJKST
+      $.
+  $}
+
+  ${
+    grpasscan2d.b $e |- B = ( Base ` G ) $.
+    grpasscan2d.p $e |- .+ = ( +g ` G ) $.
+    grpasscan2d.n $e |- N = ( invg ` G ) $.
+    grpasscan2d.g $e |- ( ph -> G e. Grp ) $.
+    grpasscan2d.1 $e |- ( ph -> X e. B ) $.
+    grpasscan2d.2 $e |- ( ph -> Y e. B ) $.
+    $( An associative cancellation law for groups.  (Contributed by SN,
+       29-Jan-2025.) $)
+    grpasscan2d $p |- ( ph -> ( ( X .+ ( N ` Y ) ) .+ Y ) = X ) $=
+      ( cgrp wcel cfv co wceq grpasscan2 syl3anc ) ADNOFBOGBOFGEPCQGCQFRKLMBCDE
+      FGHIJST $.
+  $}
+
+  ${
+    grpcominv.b $e |- B = ( Base ` G ) $.
+    grpcominv.p $e |- .+ = ( +g ` G ) $.
+    grpcominv.n $e |- N = ( invg ` G ) $.
+    grpcominv.g $e |- ( ph -> G e. Grp ) $.
+    grpcominv.x $e |- ( ph -> X e. B ) $.
+    grpcominv.y $e |- ( ph -> Y e. B ) $.
+    grpcominv.1 $e |- ( ph -> ( X .+ Y ) = ( Y .+ X ) ) $.
+    $( If two elements commute, they commute with each others inverses.  This
```

Comment:
FYI:
If two elements commute, **then** they commute with each other **'s** inverses.

("then" is for clarity)